rockfield/sass/style-child-theme-editor.scss

```diff
@@ -61,27 +63,32 @@
 .wp-block-button__link {
 	position: relative;
 
-	&:before {
+	&::before {
 		border: 2px solid;
 		content: "";
 		display: block;
 		left: 3px;
 		margin: 0;
 		position: absolute;
 		top: 3px;
-		height: calc(100% - 6px);
-		width: calc(100% - 6px);
+		height: calc(100% - 10px);
+		width: calc(100% - 10px);
 	}
 }
 
+.is-style-outline .wp-block-button__link[style*="border-radius"],
+.is-style-fill .wp-block-button__link[style*="border-radius"],
 .is-style-circular .wp-block-button__link {
-	&:before {
+
+	&::before {
 		border-radius: 100px;
 	}
 }
```

Comment:
Setting border radius to fixed `100px` always not a good idea. This fails when editor sets a radius of `8px` button appears as follows. 
<img width="104" alt="image" src="https://user-images.githubusercontent.com/1935113/178710493-8b00840a-5c1a-4853-866e-827e49855cd4.png">

With the above suggested change of inheriting the border radius, this style is not required.

Reply:
Excellent, thanks for the tip. I left the `100px` in there because it was there initially. I incorporated your suggestion :)